Tools/CISupport/ews-build/steps.py

```diff
@@ -1729,7 +1729,7 @@ class ValidateCommitterAndReviewer(buildstep.BuildStep, GitHubMixin, AddToLogMix
     descriptionDone = ['Validated commiter and reviewer']
     VALIDATORS_FOR = {
         # FIXME: This should be a bot, for now validation is manual
-        'apple': ['geoffreygaren', 'markcgee', 'rjepstein', 'jbedard', 'ryanhaddad'],
+        'apple': ['geoffreygaren', 'markcgee', 'rjepstein', 'jonwbedard', 'ryanhaddad'],
```

Comment:
do we need to do case-insensitive match while matching these names?

Reply:
Looks like we do.